.github/pull_request_template.md

```diff
@@ -0,0 +1,98 @@
+## Pull Request Template
+
+### Overview
```

Comment:
Change this section to `### PR Guidelines`, add a template spot for the user's Overview content at the bottom.